static/js/timezone_detector.js

```
@@ -0,0 +1,89 @@
+ */
+function sendTimezoneToServer(timezone) {
+    // Create form data for the request
+    const formData = new FormData();
+    formData.append('timezone', timezone);
+
+    // Get CSRF token from cookie
+    function getCookie(name) {
+        let cookieValue = null;
+        if (document.cookie && document.cookie !== '') {
+            const cookies = document.cookie.split(';');
+            for (let i = 0; i < cookies.length; i++) {
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will break the flow of the code beforehand


(lint/correctness/noUnreachable)